docs/installing_deis/quick-start.rst

```diff
@@ -10,6 +10,11 @@ These steps will help you provision a Deis cluster.
 
 .. _get_the_source:
 
+Install deisctl
```

Comment:
I think we should move this under "Get the Source". IMO it makes more logical sense to grab the source first before installing any binaries. The deisctl installation docs describe how to build it from scratch so having the source ready would make it flow better :)